migration/button_helper.py

```diff
@@ -0,0 +1,30 @@
+from django.utils.translation import gettext as _
+
+from wagtail.contrib.modeladmin.helpers import ButtonHelper
+
+
+class MigrationFailureButtonHelper(ButtonHelper):
+    btn_default_classnames = ["button-small", "icon", ]
+
+    def view_migration_failure(self, obj):
+        text = _("View migration failure")
+        return {
+            "url": "",
```

Comment:
@rafaelpezzuto https://docs.djangoproject.com/en/dev/ref/contrib/sites/#getting-the-current-domain-for-full-urls